python/tidy/servo_tidy/licenseck.py

```diff
@@ -7,97 +7,14 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+mpl = "This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/"
```

Comment:
Why put these onto a single line? AFAIK the indentation never changes, just the comment character at the start of each line. IMO this form is nicer:
```
mpl = """\
This Source Code...
""",
```